src/components/organisms/AddForm/AddForm.jsx

```diff
@@ -0,0 +1,71 @@
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+import PropTypes from 'prop-types';
+import { Icon } from '../..';
+
+import './Form.scss';
+
+export const AddForm = ({ placeholder, type }) => {
+    const submitOnEnter = (event) => {
+        if (event.key === 'Enter' && event.shiftKey === false) {
+            event.preventDefault();
+            event.target.form.dispatchEvent(
+                new Event('submit', { cancelable: true, bubbles: true })
+            );
+        }
+    };
+
+    const onSubmit = (values, action) => {
+        // TODO temporary, connect with API
+        action.resetForm({
+            values: {
+                [`${type}`]: ''
+            }
+        });
+    };
+
+    const focusOnInput = (event) => {
+        if (!event.target) {
+            event.target[0].focus();
+        }
+    };
+
+    const validation = Yup.object().shape({
+        [`${type}`]: Yup.string().required('Required')
+    });
+
+    return (
+        <div
+            className={`form form__${type}`}
+            role='directory'
+            onClick={focusOnInput}
+        >
+            <Formik
+                initialValues={{ name: type }}
+                validationSchema={validation}
+                onSubmit={onSubmit}
+            >
+                {({ handleSubmit, errors, touched }) => (
+                    <Form>
+                        <Field
+                            name={type}
+                            as='textarea'
+                            placeholder={placeholder}
+                            validate={validation}
+                            onKeyDown={submitOnEnter}
+                        />
+                        {errors.type && touched.type ? (
```

Comment:
We should just put a <Text/> component in here, and add additional type for that ~ "error"

Reply:
Also in case of that specific design, I think that we shouldn't display error message, we should for example highlight the borded of the form and change color for red for few seconds, and any error from API would be displayed via toast so yeah, displaying the error as a text is bad idea imo

Reply:
Done